File: sdk/lua/sdk/IScreen.lua
--
-- 实现screen模块时请按照这个接口的规定来
--
--

local IScreen = {}
IScreen.__index = IScreen

function IScreen:getPortSize()
    -- return screen resolution
end

function IScreen:getScreen(width)
    -- return promise/future
end

return IScreen

File: Egret/poco/sdk/IScreen.js
var IScreen = function () {
}

IScreen.prototype.getPortSize = function () {
    // return [width, height] in pixels of float type
}

IScreen.prototype.getScreen = function (width) {
    // return promisable
}
        
try {
    module.exports = IScreen;
} catch (e) {
    if (window.module && window.module.exports) {
        window.module.exports = IScreen;
    }
}